### PR TITLE
Fix conversationSignature key error

### PR DESCRIPTION
**Pull request by jacobgelling**
Here are the changes I made to get Sydney.py working for me, as discussed in #51.

### CODE REVIEW

File: sydney/sydney.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import websockets.client as websockets
 from aiohttp import ClientSession, TCPConnector
 from websockets.client import WebSocketClientProtocol
+import urllib.parse
```
Comment (vsakkas) [inline sydney/sydney.py:10]: Nitpick: Please use isort and then black in order to keep the imports (and the code in general) tidy

File: sydney/sydney.py [hunk 2]
```diff
@@ -360,7 +372,11 @@ async def start_conversation(self) -> None:
 
             self.conversation_id = response_dict["conversationId"]
             self.client_id = response_dict["clientId"]
-            self.conversation_signature = response_dict["conversationSignature"]
+            if response_dict.get("conversationSignature"):
```
Comment (vsakkas) [inline sydney/sydney.py:375]: I think this if is not needed anymore, we can keep only the code within the `else`. If it breaks, then it means the API has a breaking change which will likely require more code changes to Sydney.py

**Comment by vsakkas** [comment]
Hi @jacobgelling Thank you for contributing! Great work, I left only a couple minor comments. Once they're addressed, I'll merge and release a new build

**Comment by jacobgelling** [comment]
> Hi @jacobgelling Thank you for contributing! Great work, I left only a couple minor comments. Once they're addressed, I'll merge and release a new build
> 
> Edit: Merged so that I can release a working build faster, thank you a lot for contributing!!

Ah sorry I wasn't quick enough. Thanks for merging and writing this awesome client.

**Comment by vsakkas** [comment]
You're welcome and thank you for contributing!

**Pull request merged**